server/src/main/java/com/lunchtime/repository/RestaurantRepository.java

```diff
@@ -13,4 +14,17 @@
     Page<Restaurant> findByIsDeletedFalse(Pageable pageable);
 
     Optional<Restaurant> findByIdAndIsDeletedFalse(Long id);
+
+    /**
+     * Returns the list of restaurants that belong to concrete user (owner).
+     * It also returns all records that are NOT marked as deleted.
+     *
+     * @param id       user (owner) whose restaurants needs to be return
+     * @param pageable identifies what page needs to be returned and with what size
+     * @return the restaurant list of concrete user (owner)
+     */
+    @Query("select r from Restaurant r "
+        + "where r.personId in :id "
+        + "and r.isDeleted = false")
+    Page<Restaurant> findByOwnerRestList(Long id, Pageable pageable);
```

Comment:
I think `findRestaurantsByUserId` method name would be better.

Reply:
@narekacy We don't have `users` we have 'persons'

Reply:
It doesn't matter, let us use 'persons'

Reply:
person is general name for owner, waiter, user (can be general in some way)), ets..
"findByUser" ... it's unclear what this method returns: a list of the owner or the waiter who serves the specific rest list